### ISSUE TITLE
browser.getWindowRect is undefined in version 5.4.3

### ISSUE THREAD

**Issue by bsansouci** [webdriverio/webdriverio#3429]
**Environment (please complete the following information):**
 - **WebdriverIO version:** 5.4.3
 - **Mode:** WDIO Testrunner
 - **If WDIO Testrunner, running sync/async:** sync
 - **Node.js version:** 8.11.2
 - **NPM version:** 5.6.0
 - **Browser name and version:** Chrome 71
 - **Platform name and version:** macOS Mojave 10.14.2
 - **Additional wdio packages used (if applicable):** @wdio/spec-reporter

**Describe the bug**
I have some tests written using WebdriverIO and just upgraded to v5.4.3. I'm getting the error:
```
[chrome #0-0] browser.getWindowRect is not a function
[chrome #0-0] TypeError: browser.getWindowRect is not a function
[chrome #0-0]     at Object.getItem (getItem.js:9:64)
[chrome #0-0]     at Context.<anonymous> (mytest.test.ts:61:30)
[chrome #0-0]     at new Promise (<anonymous>)
```

If I print `browser.getWindowRect` I see it's `undefined`.

**Expected behavior**
That function should still be there given it's part of [webdriver](https://github.com/webdriverio/webdriverio/blob/dfc73b9acceac1ef59e366e106879bc904919257/packages/webdriver/protocol/webdriver.json#L218)

**Comment by abjerstedt** [comment]
Chrome is not running the webdriver API yet, its still running json web protocol. 
The command you SHOULD be using for chrome is:
https://webdriver.io/docs/api/jsonwp.html#getwindowsize

**Comment by bsansouci** [comment]
Thanks for the quick response. Just confirmed that this works.
Looking at the [status page](https://chromium.googlesource.com/chromium/src/+/master/docs/chromedriver_status.md) of chromedriver, `getWindowRect` should be implemented in the latest version (2.45 is the version we're using, which is the latest).
